server.js

```diff
@@ -68,6 +68,14 @@ app.post('/api/v1/songs', (request, response) => {
     });
 });
 
+
+app.get('/api/v1/playlists', (request, response) => {
+  database('playlists').select('playlist_name')
+    .then((songs) => {
```

Comment:
Why is this returning songs?  Shouldn't it be returning playlists?

Reply:
yikes, sorry. good catch